### DISCUSSION TITLE
Can someone explain how "Single Video" works?

### DISCUSSION THREAD

**Discussion by Nate82** [thygate/stable-diffusion-webui-depthmap-script#431] [General]
Sorry for the stupid question, bit of a noob when it comes to depth maps and this feature and I couldn't find any info about it.

What I'm trying to do is take my 6 second video (includes talking) and run it through the extension to create the same exact video with talking but the "movement" added this script does. Is this possible?

In the Single Video tab I load my video, select Generate 3D inpainted mesh and it says **"Creating mesh-videos is not supported. Please split video into frames and use batch processing."** So I uncheck that and let it run for over an hour and it gave me a depth map AVI video.

Is there any way to just upload a video and have this script add the zoom/dolly effect to it or do I have to split the video into frames and use batch process? If so, what is the purpose of the Single Video tab? Just to get a depth map from a video?

**Comment by sonakravtsenko3** [comment]
> Just to get a depth map from a video?

Indeed - these can later be used in other tools. You can also generate stereoscopic videos right away

The project needs more documentation overall...
